src/rest-client-v2.ts

```diff
@@ -1141,6 +1141,33 @@ export class RestClientV2 extends BaseRestClient {
     return this.getPrivate('/api/v2/spot/wallet/deposit-records', params);
   }
 
+  /**
+   * Upgrade Account - Upgrade to unified account mode
+   * No account type restrictions; both parent and sub-accounts are supported.
+   * This interface is only used for upgrading to the unified account mode.
+   * Please note that as the account upgrade process takes approximately 1 minute,
+   * the successful response you receive only indicates that the request has been received,
+   * and does not mean that the account has been successfully upgraded to a unified account.
+   * Please use the query upgrade status interface to confirm whether the account upgrade is successful.
+   */
+  upgradeToUnifiedAccount(params?: {
+    subUid?: string;
+  }): Promise<APIResponse<null>> {
+    return this.postPrivate('/api/v2/spot/account/upgrade', params);
+  }
+
+  /**
+   * Get Upgrade Status - Get account upgrade status
+   * No account type restrictions; both parent and sub-accounts are supported.
+   */
+  getUnifiedAccountUpgradeStatus(params?: { subUid?: string }): Promise<
+    APIResponse<{
+      status: 'processProcessing' | 'successSuccess' | 'failFailed';
+    }>
```

Comment:
```suggestion
    APIResponse<SwitchStatusResponseV3 >
```

Seems to use the same response, right? Ignoring the V3 suffix in this one case.